sys-kernel/bootengine/bootengine-9999.ebuild

```diff
@@ -10,7 +10,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="deba0732daec569545cf456f0cc514f17c7529b5"
+	CROS_WORKON_COMMIT="HEAD"
```

Comment:
Is this meant as a workaround until https://github.com/coreos/bootengine/pull/143 merges?

Reply:
@ajeddeloh Yes, once https://github.com/coreos/bootengine/pull/143 is merged this commit hash can be set.